samples/configureRetries.js

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {
+  IdempotencyStrategy,
+} = require('@google-cloud/storage/build/src/storage');
+
+/**
+ * This application demonstrates how to perform basic operations on buckets with
```

Comment:
I would get rid of lines 19-25 if possible. it seems like copy from a template that sounds too generic to be helpful.